backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -248,8 +262,8 @@ public Parcel Update(Parcel parcel)
 
             if (allowEdit)
             {
-                this.Context.Entry(originalParcel).CurrentValues.SetValues(parcel);
                 this.Context.Entry(originalParcel.Address).CurrentValues.SetValues(parcel.Address);
```

Comment:
I'm not a huge fan of these `SetValues(...)` functions as they copy all the data across and are full of risk.  But at the moment they are simple and get the job done without having tons of manual boilerplate.

Reply:
Might want to consider mapping in the future.